source/particle/world.cc

```diff
@@ -1206,9 +1206,9 @@ namespace aspect
                              "of particles. It will be checked and enforced after mesh "
                              "refinement and after particle movement. "
                              "If there are "
-                             "n_number_of_particles < min_particles_per_cell "
+                             "\\texttt{n\_number\_of\_particles} $<$ \texttt{min\\_particles\\_per\\_cell} "
```

Comment:
Oh, my apologies. I had tested everything else and that was the last error I got which I thought I could fix just by hand :-( Should have learned by now that that doesn't work :-((